Description: Enable support for python2.6
Author: Luca Falavigna <dktrkranz@debian.org>
Bug: http://bugs.debian.org/642545
Forwarded: not-needed

Index: pyparted/configure.ac
===================================================================
--- pyparted.orig/configure.ac	2011-09-24 01:05:14.616973338 +0200
+++ pyparted/configure.ac	2011-09-24 01:07:24.116977306 +0200
@@ -19,7 +19,7 @@
 dnl Red Hat Author(s): David Cantrell <dcantrell@redhat.com>
 
 m4_define(libparted_required_version, 2.3)
-m4_define(python_required_version, 2.7)
+m4_define(python_required_version, 2.6)
 
 AC_PREREQ(2.59)
 AC_INIT([pyparted], [3.6], [pyparted-devel@redhat.com])
